src/view/show_mode/editor/show_ui_widgets/cue_control.py

```diff
@@ -1,19 +1,38 @@
-from PySide6.QtWidgets import QWidget, QVBoxLayout, QToolBar, QListWidget, QInputDialog, QProgressBar
+import datetime
+from enum import Enum
+
+from PySide6.QtCore import QTimer
+from PySide6.QtWidgets import QWidget, QVBoxLayout, QToolBar, QListWidget, QInputDialog, QProgressBar, QLabel
 
 from model import UIWidget, UIPage, Filter
 from view.show_mode.editor.show_browser.annotated_item import AnnotatedListWidgetItem
 from proto.FilterMode_pb2 import update_parameter
 
+from src.fish.cue_state import CueState
+
+
+class State(Enum):
+    STOP = 1
+    PLAY = 2
+    PAUSE = 3
```

Comment:
This is also defined in `cue_state.py`. Why do you have this redundant definition here.